docs/_docs/02_features/provider-cache.md

```diff
@@ -79,18 +79,18 @@ terragrunt apply
 
 * Start a server on localhost. This is the _Terragrunt Provider Cache server_.
 * Configure Terraform instances to use the Terragrunt Provider Cache server as a remote registry:
-  * Create local CLI config file `.terragrunt-cache/.terraformrc` that concatenates the user configuration from the Terraform [CLI config file](https://developer.hashicorp.com/terraform/cli/config/config-file) with additional sections:
+  * Create local CLI config file `.terraformrc` for each module that concatenates the user configuration from the Terraform [CLI config file](https://developer.hashicorp.com/terraform/cli/config/config-file) with additional sections:
      * [provider-installation](https://developer.hashicorp.com/terraform/cli/config/config-file#provider-installation) forces Terraform to look for for the required providers in the cache directory and create symbolic links to them, if not found, then request them from the remote registry.
      * [host](https://github.com/hashicorp/terraform/issues/28309) forces Terraform to [forward](#how-forwarding-terraform-requests-through-the-terragrunt-Provider-cache-works) all provider requests through the Terragrunt Provider Cache server.
   * Set environment variables:
      * [TF_PLUGIN_CACHE_MAY_BREAK_DEPENDENCY_LOCK_FILE](https://developer.hashicorp.com/terraform/cli/config/config-file#allowing-the-provider-plugin-cache-to-break-the-dependency-lock-file) allows to generate `.terraform.lock.hcl` files based only on provider hashes from the cache directory.
      * [TF_CLI_CONFIG_FILE](https://developer.hashicorp.com/terraform/cli/config/environment-variables#tf_plugin_cache_dir) sets to use just created local CLI config `.terragrunt-cache/.terraformrc`
      * [TF_TOKEN_*](https://developer.hashicorp.com/terraform/cli/config/config-file#environment-variable-credentials) sets per-remote-registry tokens for authentication to Terragrunt Provider Cache server.
 * Any time Terragrunt is going to run `init`:
-    * Call `terraform providers lock -platform=cache_provider`. This gets Terraform to request all the providers it needs from the Terragrunt Provider Cache server.
+    * Call `terraform init`. This gets Terraform to request all the providers it needs from the Terragrunt Provider Cache server.
```

Comment:
The docs need to be updated to call out where we do hash/signature checking.

Reply:
Understood. I mentioned this in the docs.